lib/src/folding/parsers/indent.dart

```diff
@@ -5,60 +5,98 @@ import '../foldable_block.dart';
 import '../foldable_block_type.dart';
 import 'abstract.dart';
 
-/// A parser for foldable blocks from lines indentation
+/// A parser for foldable blocks from lines indentation.
 class IndentFoldableBlockParser extends AbstractFoldableBlockParser {
-  final _openBlocksLinesByIndent = <int, int>{};
+  Map<int, int> _openBlocksLinesByIndent = <int, int>{};
+  List<int?> _linesIndents = [];
```

Comment:
If you call `.clear()` on them, you can keep them `final`.

Reply:
I cant because of this error: `Unsupported operation: Cannot clear a fixed-length list`

Reply:
:(